### PR TITLE
DOC Document always inline saving elements

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/1215

### CODE REVIEW

File: en/08_Changelogs/5.3.0.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ title: 5.3.0 (unreleased)
 - [Features and enhancements](#features-and-enhancements)
   - [High-level API for converting files](#file-converter)
   - [Improve customisability of rendered images](#image-rendering)
-  - [Validation for elemental content blocks when saving individual blocks](#elemental-validation)
+  - [Validation for inline-editable elemental content blocks](#elemental-validation)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.3.0.md]:
```suggestion
  - [Validation for inline-editable elemental blocks](#elemental-validation)
```
"Content block" is a specific class, so we should just say "elemental blocks". The actual heading will need to update, as well.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/5.3.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.3.0.md]: Updated

Comment (ChloeSartorelli) [reply en/08_Changelogs/5.3.0.md]: Still need to update the actual heading

Comment (ChloeSartorelli) [reply en/08_Changelogs/5.3.0.md]: @emteknetnz Still not done?

Comment (emteknetnz) [reply en/08_Changelogs/5.3.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when code changes have been merged.

**Pull request merged**